packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -118,6 +231,28 @@ const Tabs = React.forwardRef(function Tabs(props, ref) {
   const clientSize = vertical ? 'clientHeight' : 'clientWidth';
   const size = vertical ? 'height' : 'width';
 
+  const styleProps = {
+    ...props,
+    component,
+    allowScrollButtonsMobile,
+    indicatorColor,
+    orientation,
+    vertical,
+    scrollButtons,
+    textColor,
+    variant,
+    visibleScrollbar,
+    fixed: !scrollable,
+    hideScrollbar: scrollable && !visibleScrollbar,
+    scrollableX: scrollable && !vertical,
+    scrollableY: scrollable && vertical,
+    flexContainerVertical: vertical,
```

Comment:
```suggestion
```

No need for additional additional prop that already exists. For the ones that solves some conditional it's ok :)